compute/instances.go

```diff
@@ -67,7 +67,7 @@ type Instance struct {
 	Image              string                 `json:"image"`
 	Memory             int                    `json:"memory"`
 	Disk               int                    `json:"disk"`
-	Metadata           map[string]string      `json:"metadata"`
+	Metadata           map[string]interface{} `json:"metadata"`
```

Comment:
This will be a breaking change - so we'll need to bump the major version and make notes accordingly.

Reply:
Sounds good, I'll update.